src/components/ActionBar.vue

```diff
@@ -901,29 +899,21 @@ div.bar-filter-wrapper
 <style lang="sass">
 .scroll-anim
   height: 0px !important
-  margin: 10px !important
+  margin-top: 10px !important
+  margin-bottom: 10px !important
   pointer-events: none
   opacity: 0
+main.q-page
+  padding-top: 40px
 .bar-filter-wrapper
-  margin: 40px 0 8px 0
+  margin: 0 0 8px 0
 .bar-wrapper
   position: relative
   width: 100%
   height: 88px
   background: white
   box-shadow: 0px 20px 48px rgba(0, 9, 26, 0.08), 0px 7px 15px rgba(0, 9, 26, 0.05), 0px 3px 6px rgba(0, 9, 26, 0.04), 0px 1px 2.25px rgba(0, 9, 26, 0.0383252)
   border-radius: 12px
-.bar-linear-gradient-left,
```

Comment:
🫶

Reply:
ooops! 
It was just showing appreciation for you deleting the code that doesn't need to be there <3 

<img width="280" alt="Screen Shot 2022-10-06 at 20 43 12" src="https://user-images.githubusercontent.com/2939354/194463373-45483a94-d629-49ce-9e39-81290151ab26.png">